packages/jest-circus/src/utils.ts

```diff
@@ -136,7 +136,9 @@ export const getEachHooksForTest = (test: Circus.TestEntry): TestHooks => {
 export const describeBlockHasTests = (
   describe: Circus.DescribeBlock,
 ): boolean =>
-  describe.tests.length > 0 || describe.children.some(describeBlockHasTests);
+  describe.children.some(
+    child => child.type === 'test' || describeBlockHasTests(child),
```

Comment:
I love how the short-circuiting + union type narrowing works out here, so nice :sweat_smile: